### PR TITLE
[s] Paper exploit

### PR DESCRIPTION
**Pull request by Kmc2000**
## Why It's Good For The Game

Crash server bad

## Changelog
:cl:
tweak: Reduced rate at which you can write things on paper. You probably won't notice this unless you're trying to crash the server.
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
Will this mess with printers?

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -308,15 +309,17 @@
 
 
 /obj/item/paper/Topic(href, href_list)
+	if(next_write_time > world.time) //Nsv13 possible paper exploit
```
Comment (Kmc2000) [inline code/modules/paperwork/paper.dm:312]:
```suggestion
	if(next_write_time > world.time)
```

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -308,15 +309,17 @@
...
 	..()
 	var/literate = usr.is_literate()
 	if(!usr.canUseTopic(src, BE_CLOSE, literate))
 		return
-
 	if(href_list["help"])
 		openhelp(usr)
 		return
 	if(href_list["write"])
+		next_write_time = world.time + 1 SECONDS //Nsv13 - possible paper exploit
```
Comment (Kmc2000) [inline code/modules/paperwork/paper.dm:322]:
```suggestion
		next_write_time = world.time + 1 SECONDS
```

**Comment by Jalleo** [comment]
Printers dont use topic so it shouldn't affect it.

**Pull request merged**